### ISSUE TITLE
Feature: expose/emit Vue template compiler errors

### ISSUE THREAD

**Issue by codebender828** [vue-styleguidist/vue-live#33]
Hey @elevatebart ! Thanks for the work with this package.

I'm using it on a documentation site for project of mine called Chakra UI Vue. https://chakra-ui-vue.now.sh/accordion#usage

I'd like a way to access the errors captured during the template compilation process so we can relay the error to the user. Is this possible with vue-live?

**Comment by elevatebart** [comment]
Hello @codebender828,

Great work on Crakra UI. I have been a long-running advocate of the simplicity and accessibility of its react components. Glad vue-live can help understand the project even better.

Regarding the template compilation error, I believe this  is very much possible, even easy. But for now it does not work.

Would something like this do the trick ?
https://github.com/vue-styleguidist/vue-live/pull/34

**Comment by elevatebart** [comment]
I have my answer... it does not show any errors about the template.
Will try to figure it out.

**Comment by codebender828** [comment]
Hey @elevatebart ! Thanks for the quick response! Yes something like this would be good. 👍🏽⚡

In addition to emitting the error event, it would also be good if you can create an opt-in prop to allow errors to propagate to the consumer context. What do you think?

```vue
<VueLive
  :code="code"
  :shouldPropagateTemplateErrors="true"
  @error="handleTemplateErrors"
/>
```

It would be useful in case the user only has an error boundary component around the `vue-live` component and they would like to trap/handle the errors there.

Because by default it would be good to have access to those template compilation errors.

**Comment by elevatebart** [comment]
Hello @codebender828 ,

I am afraid that if we let the error bubble up, the editor will be blocked by the error and stop functioning. I will make another Issue for it. 

See if I can integrate it.

**Comment by codebender828** [comment]
Thanks! Cheers! @elevatebart !
